### PR TITLE
Configure prefetch counts

### PR DESCRIPTION
**Pull request by danielglauser**

### CODE REVIEW

File: src/scheduled_message_works/queue.clj [hunk 1]
```diff
@@ -11,39 +11,40 @@
...
+                      (get queues "scheduled-message-works.message.read")
+                      channels/read-message-requests
+                      channels/read-message-responses)
+        search-channel (wire-up/incoming-service
+                        connection
+                        "schedule-message-works.message.search"
+                        (get queues "schedule-message-works.message.search")
+                        channels/search-messages-requests
+                        channels/search-messages-responses)]
+    (let [incoming-events [ok-channel schedule-channel delete-channel read-channel search-channel]
```
Comment (cap10morgan) [inline src/scheduled_message_works/queue.clj]: These should be in `incoming-services` not `incoming-events`.

File: src/scheduled_message_works/queue.clj [hunk 2]
```diff
@@ -66,6 +68,14 @@
             "ballot-scout.ballot.delivery-update"
             channels/ballot-delivery-update)]]
 
+      ;; Set the prefetch number to the number of threads.
+      ;; Not sure if this is optimal but seems like a good place to start
+      (lb/qos ok-channel 2)
+      (lb/qos schedule-channel      (get-in config [:threads :schedule-message]))
+      (lb/qos delete-channel        (get-in config [:threads :delete-message]))
+      (lb/qos reach-channel-channel (get-in config [:threads :read-message]))
```
Comment (cap10morgan) [inline src/scheduled_message_works/queue.clj]: Tests failed on this one. Should be `read-channel`.

File: src/scheduled_message_works/queue.clj [hunk 3]
```diff
@@ -11,40 +12,40 @@
...
+                      (get queues "scheduled-message-works.message.read")
+                      channels/read-message-requests
+                      channels/read-message-responses)
+        search-channel (wire-up/incoming-service
+                        connection
+                        "schedule-message-works.message.search"
+                        (get queues "schedule-message-works.message.search")
+                        channels/search-messages-requests
+                        channels/search-messages-responses)]
+    (let [incoming-services [ok-channel schedule-channel delete-channel read-channel search-channel]
```
Comment (cap10morgan) [inline src/scheduled_message_works/queue.clj]: I'm fine with either but you'll need to:
a) add `incoming-events []` here OR
b) remove `incoming-events` on line 104

**Comment by danielglauser** [comment]
Thanks for the review @cap10morgan. We can talk about how to roll this out once I've finished testing locally.

**Comment by danielglauser** [comment]
Confirmed that this works locally, I'm able to see scheduled messages go out. This is ready to deploy.

**Pull request merged**